### PR TITLE
Navbar into develop

### PR DESCRIPTION
**Pull request by DaanEKR**
# MD-05: As a PO i want a navbar

## Tasks
Added navbar.

## Ready for Review?
- You need 2 thumbs off.
- Build (code and design) are according to Definition Of Done (see dod.md).

### CODE REVIEW

File: front-dex/src/assets/scss/helpers/_vars.scss [hunk 1]
```diff
@@ -1,7 +1,8 @@
 //main colors
-$background: #5e318f;
 $pink: #c82d8c;
 $text: #ffffff;
+$navbar: #5e318f;
```
Comment (Danny-Mus) [inline front-dex/src/assets/scss/helpers/_vars.scss]: 'primary-theme'

File: front-dex/src/assets/scss/helpers/_vars.scss [hunk 2]
```diff
@@ -1,7 +1,8 @@
...
+$navbar-hover: #68448f;
```
Comment (Danny-Mus) [inline front-dex/src/assets/scss/helpers/_vars.scss]: 'primary-theme-hover'

File: front-dex/src/app/app.component.scss [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        justify-content: flex-start;
+        align-items: center;
+        width: 100%;
+        list-style-type: none;
+
+        &-item {
+            font-family: $overpass;
+            height: 75px;
+            width: 75px;
+            padding: 0 5px 0 5px;
```
Comment (kmensink95) [inline front-dex/src/app/app.component.scss]: this could be padding: 0 5px; right?

File: front-dex/src/app/app.component.html [hunk 4]
```diff
@@ -0,0 +1,15 @@
+<nav class="navigation">
+    <ul class="navigation__list">
+        <li class="navigation__list-item">
```
Comment (Danny-Mus) [inline front-dex/src/app/app.component.html]: change to `navigation__item`

**Pull request merged**